tests/unit/lib/foxbox/event-dispatcher_test.js

```diff
@@ -0,0 +1,423 @@
+import EventDispatcher from 'js/lib/foxbox/event-dispatcher';
+
+describe('EventDispatcher >', function () {
```

Comment:
It's just an idea.
That would result in a more readable file, but that's debatable.